### PR TITLE
Use syntax highlighting in the install page.

### PR DESCRIPTION
**Pull request by bdice**
This updates the install page (and a few other cases) to use syntax highlighting in code snippets.

### CODE REVIEW

File: maintainers/datasets.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ Finally, if neither the build option nor the environment variable are set, tests
...
 Test developers can use the RAPIDS dataset locating utilities by calling the following function:
-```
+```cpp
 const std::string& get_rapids_dataset_root_dir()
 ```
 `get_rapids_dataset_root_dir()` will return a `std::string` reference to the root directory containing the datasets, as determined by the user-accessible options described above, or a default value of `/datasets` if the user did not use the options.
 
 ### Example API usage
-```
+```cpp
```
Comment (jarmak-nv) [inline maintainers/datasets.md:59]: I worry this leads to a bit low contrast for these code boxes. Contrast ratio is ~2.5 and recommended is 4.5+. https://webaim.org/resources/contrastchecker/

No idea how customizable these code formats are though (I assume non-customizable.) 

image
image

**Review by jarmak-nv — COMMENTED** [review]
Looks great overall - thanks! 

Just want to talk through my comment contrast for `cpp` format highlighting before approving.

File: maintainers/datasets.md (hunk 1 above)
Comment (bdice) [reply maintainers/datasets.md:59]: Updated! Try again:
https://deploy-preview-494--docs-rapids-ai.netlify.app/maintainers/datasets/#writing-tests-that-use-datasets
https://deploy-preview-494--docs-rapids-ai.netlify.app/install#wsl2-docker

### Names and Parentheses
image

### Operators
image

Comment (jarmak-nv) [reply maintainers/datasets.md:59]: Love it - thank you!

**Review by jarmak-nv — APPROVED** [review]

**Pull request merged**